src/Hosting/Hosting/test/HostingApplicationTests.cs

```diff
@@ -87,6 +87,37 @@ public void CreateContextCreatesNewContextIfNotUsingDefaultHttpContextFactory()
             hostingApplication.DisposeContext(context, null);
         }
 
+        [Fact]
+        public void IActivityFeatureIsCreated()
+        {
+            using var listener = new ActivityListener
+            {
+                ShouldListenTo = activitySource => (activitySource.Name == "Microsoft.AspNetCore" ||
+                                                    activitySource.Name == "DummySource"),
```

Comment:
Can we use a random name so this doesn't accidently cause issues when this test gets copied down the road?